plugin/evm/block.go

```diff
@@ -124,6 +130,21 @@ func (b *Block) handlePrecompileAccept(rules *params.Rules, sharedMemoryWriter *
 		}
 	}
 
+	// If Warp is enabled, add the block hash as an unsigned message to the warp backend.
```

Comment:
This is the simplest way to add the warp message into the warp backend. Alternatively, we can update the warp backend to provide it access to accepted blocks.

To make that change, we should add a separate handler for fetching signed block hashes so that they can be fetched by `blockHash` instead of the `UnsignedWarpMessageID`.

Reply:
I think this would be a good change for a follow-up PR, since it works as is.

Reply:
👍